.pre-commit-config.yaml

```diff
@@ -4,3 +4,7 @@ repos:
     hooks:
       - id: clang-format
         args: ["--style", "Google", "src/**"]
+  - repo: https://github.com/hhatto/autopep8
```

Comment:
This bit installs the autopep8 linter, you can test it by running

```
pre-commit install
```

After you run

```
python3 -m pip install '.[dev]' 
```

from the project root